mapiproxy/servers/default/emsmdb/dcesrv_exchange_emsmdb.c

```diff
@@ -281,383 +281,17 @@ static enum MAPISTATUS dcesrv_EcDoDisconnect(struct dcesrv_call_state *dce_call,
-                                if (notification->parameters.object_parameters.tag_count
-                                    && notification->parameters.object_parameters.tag_count != 0xffff) {
-                                        reply->NotificationData.FolderCreatedNotification.NotificationTags.Tags = talloc_memdup(mem_ctx, notification->parameters.object_parameters.tags, notification->parameters.object_parameters.tag_count * sizeof(enum MAPITAGS));
-                                }
-                                else {
-                                        reply->NotificationData.FolderCreatedNotification.NotificationTags.Tags = NULL;
-                                }
-                                break;
-                        case MAPISTORE_OBJECT_MODIFIED:
-                                reply->NotificationData.FolderModifiedNotification_10.FID = notification->parameters.object_parameters.object_id;
-                                reply->NotificationData.FolderModifiedNotification_10.TagCount = notification->parameters.object_parameters.tag_count;
-                                if (notification->parameters.object_parameters.tag_count
-                                    && notification->parameters.object_parameters.tag_count != 0xffff) {
-                                        reply->NotificationData.FolderModifiedNotification_10.NotificationTags.Tags = talloc_memdup(mem_ctx, notification->parameters.object_parameters.tags, notification->parameters.object_parameters.tag_count * sizeof(enum MAPITAGS));
-                                }
-                                else {
-                                        reply->NotificationData.FolderModifiedNotification_10.NotificationTags.Tags = NULL;
-                                }
-
-                                if (notification->parameters.object_parameters.new_message_count) {
-                                        reply->NotificationData.FolderModifiedNotification_1010.TotalMessageCount = notification->parameters.object_parameters.message_count;
-                                }
-
-                                break;
-                        default: /* MAPISTORE_OBJECT_DELETED */
-                                reply->NotificationData.FolderDeletedNotification.ParentFID = notification->parameters.object_parameters.folder_id;
-                                reply->NotificationData.FolderDeletedNotification.FID = notification->parameters.object_parameters.object_id;
-                        }
-                }
-        }
-
-end:
-	*sizep += libmapiserver_RopNotify_size(mapi_repl);
-
-	return success;
-}
-
 static struct mapi_response *EcDoRpc_process_transaction(TALLOC_CTX *mem_ctx, 
 							 struct emsmdbp_context *emsmdbp_ctx,
-							 struct mapi_request *mapi_request)
+							 struct mapi_request *mapi_request,
+							 struct GUID uuid)
```

Comment:
`uuid` is not used but `emsmdbp_ctx->session_uuid` (and how this function is called right now they are the same)